src/microscope_autofocus/python/autofocus.py

```diff
@@ -100,43 +139,38 @@ def find_best_focus(
         cv2.destroyAllWindows()
 
 
-SHOW_STEP_INFO = False
-SHOW_STEP_IMAGES = False
-
-focus_scores: list[float] = []
```

Comment:
I have mixed feelings about removing the globals. On the one hand I understand the principle behind it, but logging I think is an area where global flags like this are generally accepted (when was the last time you saw a method with log_level as a param?). The point of this code is really to show how to do auto-focus, and it feels like all the extra params this change adds distracts from that more than it clarifies any of the logic. In retrospect, I wish I'd use proper logging utilities for this, but that seems like a bigger change than should happen to this example without running it and making sure nothing broke.

Reply:
That's fair. I can back those changes out. Would you prefer that I make those as separate fixup commits or just amend the existing commits?

Reply:
Ok. I'd prefer to amend my commits then. I'll let you know when that's done.

Reply:
Ok, I've pushed the changes:  globals should be untouched now.